### PR TITLE
Create newcomer resources

### LINKED ISSUE #221: Create Newcomers Resources Page
## Description
As a contributor,
I need create a newcomers Resources Page,
so that I can help newcomers get some good resources to get started

## Mocks
screen shot 2018-06-27 at 1 12 23 am

### Update [Required]
- [ ] newcomers.component.html
- [ ] newcomers.component.ts

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
8 hours 

@ancyp @Janiceilene Can you please assign me the issue? :)

### PR DESCRIPTION
**Pull request by BaaniLeen**
### Description
Created a newcomers resources page.

Fixes #221 

Can be merged only after #212 

### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation
  

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Has been tested on localhost.

### Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged 
  

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: src/app/newcomers/nonCodeMethodsData.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    {
+        methodName: 'Quality Assurance',
+        image: '/assets/img/qualityAssurance.jpg',
+        details: 'Finding and verifying bugs is an important part of software development.'
+    },
+    {
+        methodName: 'Other',
+        image: '/assets/img/others.jpg',
+        // tslint:disable-next-line:max-line-length
+        details: 'If you have an idea that may help an organization, reach out to them and let them know! Sometimes the best task is one that the organization hasn’t even considered yet.iil/'
```
Comment (sara-02) [inline src/app/newcomers/nonCodeMethodsData.ts]: `iil/` possible typo?

Comment (BaaniLeen) [reply src/app/newcomers/nonCodeMethodsData.ts]: Yes. Thanks for pointing it out. @sara-02 :)

**Comment by BaaniLeen** [comment]
@ancyp @Tharangi Resolved the conflicts. :)

**Comment by ancyp** [comment]
Two issues to be opened:
Cross over text in modal
image
Buttons need to be aligned.

**Pull request merged**